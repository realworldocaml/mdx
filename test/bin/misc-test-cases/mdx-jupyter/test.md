## Generating a Jupyter notebook

```ocaml
let x = 1
let () = assert true
```

This block should not be executed by the tests:
```ocaml skip
let x = 2
let () = assert false (* Don't print this! *)
```

```ocaml
let () = print_int x
```
