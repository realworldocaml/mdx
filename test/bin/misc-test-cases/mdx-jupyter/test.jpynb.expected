{"metadata":{"kernelspec":{"display_name":"OCaml 4.07.1","language":"OCaml","name":"ocaml-jupyter"},"language_info":{"name":"OCaml","version":"4.07.1","codemirror_mode":"text/x-ocaml","file_extension":".ml","mimetype":"text/x-ocaml","nbconverter_exporter":null,"pygments_lexer":"OCaml"}},"nbformat":4,"nbformat_minor":2,"cells":[{"cell_type":"markdown","metadata":{},"source":"## Generating a Jupyter notebook\n\n\n\n"},{"cell_type":"code","metadata":{},"source":"\nlet x = 1\nlet () = assert true\n","outputs":[],"execution_count":1},{"cell_type":"markdown","metadata":{},"source":"\n\nThis block should not be executed by the tests:\n"},{"cell_type":"code","metadata":{},"source":"\nlet x = 2\nlet () = assert false (* Don't print this! *)\n","outputs":[],"execution_count":2},{"cell_type":"markdown","metadata":{},"source":"\n"},{"cell_type":"code","metadata":{},"source":"\nlet () = print_int x\n","outputs":[],"execution_count":3}]}